src/scripts/main.js

```diff
@@ -354,7 +354,22 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM. If it doesn't, `document.querySelector('.dashboard')` will return `null`, leading to an error when attempting to append rows to `table`. Consider adding a check to ensure `table` is not `null` before proceeding with the loop.